app-drawer/app-drawer.html

```diff
@@ -311,9 +336,22 @@
         this._setPosition(this.align);
       },
 
+      _focusHandler: function(event) {
+        if (this._isAboveContent() && !this.contains(Polymer.dom(event).rootTarget)) {
```

Comment:
I actually don't need to cross ShadowDOM boundaries, since there are no focusable elements in the LocalDOM and all I'm interested in elements in the LightDOM.